include/rle_data.hpp

```diff
@@ -326,7 +326,7 @@ namespace Gamera {
 	m_pos--;
 	if (!check_chunk()) {
 	  if (m_i != m_vec->m_data[m_chunk].begin()) {
-	    iterator prev_i = prev(m_i);
+	    iterator prev_i = std::prev(m_i);
```

Comment:
Apparently, these functions have been introduced into C++ 2011. That's why the function definition is activated only #ifdef __cplusplus < 201103. I will add the correct namespace (std) and merge your changes into the main branch. Thanks!